pkg/e2e/infra/infra.go

```diff
@@ -34,10 +35,15 @@ var _ = g.Describe("[Feature:Machines] Managed cluster should", func() {
 		client, err := e2e.LoadClient()
 		o.Expect(err).NotTo(o.HaveOccurred())
 
-		machines, err := getMachines(client)
+		machineList := mapiv1beta1.MachineList{}
```

Comment:
can we update getMachines to use `client.List` and keep this as it was?

Reply:
I removed `getMachines` function on purpose. As long as it can be replaced by a single invocation, we should not hide it in another function/method.